### ISSUE TITLE
Style Details/Summary like accordian

### ISSUE THREAD

**Issue by dubcanada** [picturepan2/spectre#256]
Hello,

I think we should style the default details/summary elements and use that as the accordion instead of custom divs with JS.

Thoughts?

**Comment by picturepan2** [comment]
@dubcanada Any additional style needed?

**Comment by chrisssycollins** [comment]
@dubcanada You could use existing classes for the most part but will need to remove the default arrow and add the `transform` to the new arrow as it doesn't look for `input:checked`.  Also I don't think there is any need for `.accordion-body` when using `details/summary`.

```html
<div class="accordion">
    <details class="accordion-item">
        <summary class="accordion-header"> <i class="icon icon-arrow-right mr-5"></i> Copyright 1999-2014.</summary>
        <p>It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum</p>
    </details>
</div>
```
_accordions.scss
```scss
// Accordions
.accordion {
    .accordion-item {
        input {
            &:checked {
                &~.accordion-header {
                    .icon {
                        transform: rotate(90deg);
                    }
                }
                &~.accordion-body {
                    max-height: 50rem;
                }
            }
        }

        .accordion-header {
            outline: none;
            cursor: pointer;
            display: block;
            padding: $unit-1 $unit-2;

            .icon {
                transition: all .2s ease;
            }
        }

        .accordion-body {
            margin-bottom: $layout-spacing;
            max-height: 0;
            overflow: hidden;
            transition: max-height .2s ease;
        }
    }

    .accordion-header::-webkit-details-marker {
        display: none;
    }
    .accordion-item[open] {
        .accordion-header {
            .icon {
                transform: rotate(90deg);
            }
        }
    }
}
```
I'd happily create a PR but not sure if @picturepan2 want's it in?

**Comment by picturepan2** [comment]
@chrisssycollins I am exploring the details/summary elements. Please check out to see if any improvement is needed once I commit. Thanks.

**Comment by picturepan2** [comment]
Released in v0.4.1
